src/libs/utils.ts

```diff
@@ -105,9 +105,9 @@ export const formatNumber = new Intl.NumberFormat('en-US').format
 
 export const workspaceAccessLevels = ['NO ACCESS', 'READER', 'WRITER', 'OWNER', 'PROJECT_OWNER']
 
-export const hasAccessLevel = _.curry((required, current) => {
+export const hasAccessLevel = (required, current) => {
```

Comment:
Should type definitions for the function interface be added here?

```suggestion
export const hasAccessLevel = (required: string, current: string): boolean => {
```

Probably have to use string for now, but in the future we could have a type for access levels.

Reply:
agree, feel like its a bit out of the scope of this ticket though

Reply:
To clarify, I was only asking for `string` types in this PR. Since this PR changes this function by removing curry for (I assume) type flow, I think it's reasonable to add those types here. Types for access level is definitely out of scope.